webapp/src/main/webapp/iaf/gui/js/services.js

```diff
@@ -1049,6 +1049,8 @@ angular.module('iaf.beheerconsole')
 								if(appConstants.init == 1) {
 									if(rejection.config.headers["Authorization"] != undefined) {
 										console.warn("Authorization error");
+									} else {
+										Toastr.error("Failed to connect to backend", "CONNECTION REFUSED");
```

Comment:
Dit is een aanname van het type 'Uw sessie is verlopen omdat u te lang gewacht heeft'.
Als er niets is waar je iets mee kan, kan je hoogstens zeggen dat je geen antwoord hebt gekregen.

Reply:
`CONNECTION REFUSED` is toch geen antwoord?

Reply:
Connection refused is wel een antwoord, de eerste portier zegt: je mag niet naar binnen.